src/database/migrations/0001_create_canonical_fields.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE canonical_fields (
+  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  label VARCHAR NOT NULL,
+  short_code VARCHAR NOT NULL,
+  data_type VARCHAR NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
+);
+
+comment on table canonical_fields is
+  'Canonical fields are those fields that are unique across organizations.';
+comment on column canonical_fields.label is 'A word or phrase that communicates the meaning of the field for end users';
+comment on column canonical_fields.short_code is 'An externally meaningful alphanumeric code tht can be used to reference this field without knowing the internal id';
```

Comment:
The comment on `short_code` helps me a lot. But then it raises a question: can we make `short_code` unique? Why or why not? You probably answered this somewhere already but I am still not sure I grasped it.

Reply:
Oh snap I actually meant to add that -- yes, we should.  I'll include that in this migration

Reply:
:+1: